### DISCUSSION TITLE
Migration está criando tabela em banco errado

### DISCUSSION THREAD

**Discussion by MelloRenato** [balta-io/forum#518] [Tira Dúvidas] [answered]
Curso: Fundamentos do Entity Framework
Aula:  Atualizando o banco

Estou usando .Net 8.

Criei meu models, criei a migration (dotnet ef migrations add Inicial) e fiz o Update do banco.

Tudo foi executado sem erro porém, eu gostaria que criasse um banco chamado "RMFinanca" e ao invés disso as tabelas foram criadas no banco "Master" que já existia.
![image](https://github.com/balta-io/forum/assets/12654234/83c1902f-9fc0-423c-9a0e-e934dd0d62f4)

Pelo que compreendi o banco deveria ser criando de acordo com DataContext, certo?
![image](https://github.com/balta-io/forum/assets/12654234/fbc2f139-b4b7-4c89-85da-8b9b6b81ce76)

O que fiz de errado?

Grato,
MelloRenato

**Comment by malighieri** [comment, marked as answer]
Na string de conexão do EF o item que se refere ao nome do banco se chama _**Initial Catalog**_.
Sendo assim, para que o banco tenha o nome _RMFinanca_ sua string de conexão deveria ser:
_"Server=localhost;Initial Catalog=RMFinanca;User ID=sa;Password=..._

**Reply by MelloRenato** [reply]
Obrigado.

Vou testar.

Grato
